### ISSUE TITLE
Improve Gammapy conda install docs

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#1007]
In #994 the install docs were improved a bit, they now look like this:
https://gammapy.readthedocs.io/en/latest/install/index.html

The next steps are to introduce an `environment.yml` for Gammapy as already mentioned in
https://gammapy.readthedocs.io/en/latest/install/index.html#development-version
and to improve the conda install docs here:
https://gammapy.readthedocs.io/en/latest/install/conda.html

This content should be moved over to the Sphinx docs, so that we don't maintain installation / getting started instructions in the notebooks in parallel:
https://nbviewer.jupyter.org/github/gammapy/gammapy-extra/blob/master/index.ipynb#Set-up
https://nbviewer.jupyter.org/github/gammapy/gammapy-extra/blob/master/notebooks/tutorial_setup.ipynb

ctapipe has this:
https://cta-observatory.github.io/ctapipe/getting_started/index.html#step-4-set-up-your-package-environment
but what is missing is how to get the `environment.yml` for people that just want to get the stable version, and never clone the code repo!?

I'm putting this under the v0.7 milestone.

@joleroi - You or me?

**Comment by cdeil** [comment]
I wanted to try out the conda 0.6 and did `conda install gammapy`, which did fetch v0.6 but some tests fail:
https://gist.github.com/cdeil/2e492b1788c6c628fbec22de887ec315

Apparently what happened was that I had an older version of Gammapy installed, and then conda just added the files for 0.6, but didn't remove the previous install, so there's a mix of old and new files. This is really bad, many users will run into this issue.

Can someone using conda (@bsipocz? @joleroi? @adonath?) please take a stab at 
http://docs.gammapy.org/en/latest/install/conda.html
and add the proper command to update with conda?

Or let me know via a comment here, and I'll try to update that docs page tomorrow before sending the Gammapy 0.6 release announcement.

**Comment by bsipocz** [comment]
I'm not using conda myself, but it seems that the page you linked actually talks about updating to the latest version with ``conda update gammapy``. Maybe what you want is to reorder and rehighlight things, move the update to the top and have the install from scratch the second item?

**Comment by cdeil** [comment]
As far as I know `conda install` is equivalent to `pip install --update`. Yes / no?
E.g. here they also recommend `conda install` and `pip install --update` to update to a new release:
http://matthewrocklin.com/blog/work/2017/05/08/dask-0.14.3

**Comment by bsipocz** [comment]
Maybe @astrofrog or @mwcraig can comment on it, as I don't use conda beyond CI.

**Comment by cdeil** [comment]
We have new install instructions here:
http://docs.gammapy.org/dev/getting-started.html
Basically we tell people to use conda and we offer a stable env for each version we plan to release (at least, could have any number of configs).

Closing this issue now. Comments still welcome of course. If you suggest changes / actions, a new issue is preferred.
